typeguard.py

```diff
@@ -482,9 +499,13 @@ def typechecked(func: Callable = None, *, always: bool = False):
         warn('no type annotations present -- not typechecking {}'.format(function_name(func)))
         return func
 
+    frame = inspect.stack()[1][0]
+
+    dec_later = _inject_forward_refs(frame, func)
```

Comment:
This looks strange to me for two resons.

*  ``dec_later`` appears not to be used anywhere else
* ``_inject_forward_refs`` does not return anything

Reply:
sry copy pasta :(